src/LuaPrinter.ts

```diff
@@ -126,7 +126,7 @@ export class LuaPrinter {
             if ((this.options.luaLibImport === LuaLibImportKind.Require && luaLibFeatures.size > 0)
                 || this.options.luaLibImport === LuaLibImportKind.Always)
             {
-                header += `require("lualib_bundle");\n`;
+                header += `require("lualib_bundle")\n`;
```

Comment:
What if the first line after this starts with a parenthesis?

Reply:
Nice catch!